### PR TITLE
A user should see different text to create the first todo item

### PR DESCRIPTION
**Pull request by vlasiak**
Displaying another link text in case there are no items in a list.

### CODE REVIEW

**Comment by vlasiak** [comment]
@sergeyradkevich, please verify and review. Link to staging server is [here](https://tada-project.herokuapp.com).

**Comment by sergeyradkevich** [comment]
@vlasiak, will take a look in a few hours right after the my today's meetings.

**Comment by sergeyradkevich** [comment]
@vlasiak, Tested, LGTM.

File: app/assets/javascripts/to_do_items.coffee [hunk 1]
```diff
@@ -35,9 +35,13 @@ class @TodoItem
...
+  renameLink = (id, name) ->
+    detectAddLink(id).text(name)
+
   renderItem = (options) ->
     removeCallout options.listId
     appendNewOne options.listId, options.html
+    renameLink options.listId, options.newLinkName
     focusOn options.listId
     highlightNewOne options.id if options.id
     checkOnClick detectCheckbox(options.id)[0]
```
Comment (sergeyradkevich) [inline app/assets/javascripts/to_do_items.coffee:47]: @vlasiak, Just FYI: `renderItem` method does more than just rendering of the item, so I would decompose it in the future once there are some changes to be done.

File: app/models/list.rb [hunk 2]
```diff
@@ -14,4 +14,8 @@ def incompleted_items
     items.reject { |item| item.done? }
   end
 
+  def has_items?
+    !items.empty?
+  end
```
Comment (sergeyradkevich) [inline app/models/list.rb:19]: @vlasiak Another way to do this without inversion is to use `present?` See [the details](http://apidock.com/rails/Object/present%3F). Also, please make sure that it doesn't make any queries into DB when it is called.

**Comment by sergeyradkevich** [comment]
@vlasiak, Thanks for moving to `haml` :+1:

File: app/views/to_do_items/_new.html.haml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+= form_for @item, url: to_do_items_path, method: :post, remote: true, html: {id: "form-#{@item.list_id}"} do |f|
+  .form-group
+    = f.text_field :text, id: "item-input-#{@item.list_id}", class: 'text-input', placeholder: "#{t 'item.add.placeholder'}"
+    = f.hidden_field :list_id, value: @item.list_id
+
+  = f.submit "#{t 'item.add.button'}", id: "add-item-button-#{@item.list_id}", class: 'btn btn-success btn-sm'
+  = f.button "#{t 'item.cancel_button'}", id: "cancel-item-button-#{@item.list_id}", class: 'btn btn-default btn-sm', type: :button
```
Comment (sergeyradkevich) [inline app/views/to_do_items/_new.html.haml]: @vlasiak, I'm wondering whether the following would work:

``` ruby
f.submit t('item.add.button')
```

If it does, can you please get rid of `#{}` expression through this PR?

Comment (sergeyradkevich) [inline app/views/to_do_items/_new.html.haml]: I mean where `#{}` is being used along with `t` method.

**Comment by sergeyradkevich** [comment]
@vlasiak, Reviewed.

File: app/models/list.rb (hunk 2 above)
Comment (vlasiak) [inline app/models/list.rb:19]: Thanks, no any DB queries.

**Comment by vlasiak** [comment]
@sergeyradkevich, updated.

**Comment by sergeyradkevich** [comment]
@vlasiak, Two questions:
1. Have you redeployed the version on [staging server](https://tada-project.herokuapp.com/)?
2. I suppose the tests pass locally and the Circle CI warns just because the configuration is in progress, right?

**Comment by vlasiak** [comment]
@sergeyradkevich , 

> Two questions...

According to this comment:
1. Redeployed it few minutes ago. [Link](https://tada-project.herokuapp.com/).
2. That's right, all the tests passed locally, Circle CI isn't configured yet.

**Pull request merged**